unit-test/addTask.js

```diff
@@ -0,0 +1,11 @@
+const addTask = (task) => {
+  const taskCopy = task;
+
+  taskCopy.push({
+    completed: false,
+    index: taskCopy.length + 1,
+    description: taskCopy,
+  });
+};
```

Comment:
- [ ] The main purpose of writing unit tests is to make sure the real functions are tested. Kindly ensure not a create another function, rather test the main function responsible for adding the tasks